_i18n/en.yml

```diff
@@ -908,7 +908,7 @@ specs:
   block_emission_title: Emission curve
   block_emission_pre: "To make sure there will always be an incentive to mine Monero and keep it safe, the emission is infinite. There are two main emissions:"
   block_emission_main: "first, main curve: ~18.132 million coins by the end of May 2022"
-  block_emission_tail: "then, tail curve: 0.6 XMR per 2-minute block, kicks in once main emission is done, translates to <1% inflation decreasing over time"
+  block_emission_tail: "then, tail curve: 0.6 XMR per 2-minute block, kicked in after main emission is done, translates to <1% inflation decreasing over time"
```

Comment:
```suggestion
  block_emission_tail: "then, tail emission (0.6 XMR per 2-minute block) kicked in after the main emission was completed. This translates to <1% inflation, with the percentage decreasing over time"
```

Reply:
Missed this, thanks. could rephrase to slip "zero" in there:  "with the percentage trending toward zero"

Reply:
Edited